### DISCUSSION TITLE
PLUSWIND wind energy database by LBNL (not mine!)

### DISCUSSION THREAD

**Discussion by williamhobbs** [blaylockbk/Herbie#252] [Show and tell]
[_note at the beginning: I had nothing to do with this application, but I figured "show and tell" might be the best discussion category. Feel free to move/remove/relabel/etc._]

Researchers at Lawrence Berkely National Lab, Dev Millstein, Seongeun Jeong, Amos Ancell (@AmosRAncell), and Ryan Wiser, recently published a paper [1] about a dataset [2] that used Herbie, among other tools, for modeling existing wind farms. They compared three wind resource data sources, ERA5, MERRA-2, and HRRR (using Herbie) against reported actual power generation data. 

The code they used is available here: https://github.com/AmosRAncell/PLUSWIND. 

[1] https://doi.org/10.1038/s41597-023-02804-w
[2] https://a2e.energy.gov/project/pluswind

**Comment by blaylockbk** [comment]
This is awesome. Thanks for sharing 😊
